Update README (#15)
makinacorpus/caue24: README.md

# General Description

Open source code of caue24 website https://album-des-territoires.cauedordogne.com

> <ici mettre description du site>

Reviewer: ```suggestion
<!-- Put here web site description -->
```

This website is mostly interactive maps loading static data (markdown and geojson) and some external tiles.

# Architecture

Static website build through Jekyll.
Published on github pages, using jekyll from github pages.

## Technologies

* static html

> * leaflet

Reviewer: ```suggestion
* [Leaflet](https://leafletjs.com/) v0.6.4
```

> * backbone js

Reviewer: ```suggestion
* [BackboneJS](https://backbonejs.org/) v1.0.0
```

> * turfjs

Reviewer: ```suggestion
* [TurfJS](https://turfjs.org/)
```

> # installation

Reviewer: ```suggestion
# Installation
```

## Development

Setup a Jekyll as the one used by github, a simple jekyll should be enough.

> Then run it frmo the source code with:

Reviewer: ```suggestion
Then run it from the project directory with:
```

```
  jekyll serve --watch
```

>  Then browse to: http://localhost:4000/```

Reviewer: ```suggestion
 Then browse to: `http://localhost:4000/`
```

## Deployments

Production site is automatically updated by github pages's jekyll when some code is pushed on branch `gh-pages`.

> ## architecture fonctionelle

Reviewer: ```suggestion
## Functional architecture
```

> (pour le futur dev qui viens corriger)

Reviewer: ```suggestion
<!-- (pour le futur dev qui viens corriger) -->
```
